example/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
And inside `example/build/libs/example.jar`, there are two `.java` files: `$AutoValue_Person.java` and `AutoValue_Person.java`.